test/integration/scene-admin/remove-scene-admin-handler.spec.ts

```diff
@@ -148,16 +146,14 @@ test('DELETE /scene-admin - removes administrator access for a scene', ({ compon
       owner
     )
 
-    expect(response.status).toBe(400)
+    expect(response.status).toBe(401)
```

Comment:
Should we return a 404 when trying to remove a non-existent admin?

Reply:
hummm maybe better a 403 in that case, WDYT?